### PR TITLE
added visualize query

### PR DESCRIPTION
**Pull request by ajayshekar**

### CODE REVIEW

File: services/config.rb [hunk 1]
```diff
@@ -103,10 +103,17 @@
...
       relates_to @filter(has(user))
     }
-    query(func: has(group)) @filter(NOT uid(g)) {
+    invalid_items as query(func: <%= filter['group'] %>) @filter(NOT uid(g)) {
       <%= default_predicates %>
     }
-  }
+    
+    visualize(func: uid(invalid_items)) {
+      <%= default_predicates %>
```
Comment (armaneous) [inline services/config.rb]: `default_predicates` is only meant to be a baseline, something we're guaranteed all nodes have. There should be more predicates that we try to expand if we can. In this case, I think there are plenty of `group` properties that would be good to know if it's an unused IAM group.

Comment (armaneous) [reply services/config.rb]: Need to change from approve to request changes... requires new comment.

Comment (ajayshekar) [reply services/config.rb]: Gotcha! Will add in some useful predicates. I left it as `default_predicates` because the query just returned the same.

Comment (armaneous) [reply services/config.rb]: Then it's likely that both need to be updated

**Pull request merged**